### DISCUSSION TITLE
can't run obfuscated JAR

### DISCUSSION THREAD

**Discussion by MeteorTony** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#66] [Q&A] [answered]
I run the command "java --enable-preview -jar PA1-obf.jar [--unicode] puzzles/00-unicode-test.game" in the terminal located in the artifacts directory. But here's the error:
<img width="917" alt="Screenshot (7)" src="https://user-images.githubusercontent.com/61377153/135745818-0d15c55a-fbe5-4a88-b4db-0999e7cd7c2e.png">
what can I do to solve it?

**Comment by Derppening** [comment, marked as answer]
`[--unicode]` is to indicate that you may optionally provide the `--unicode` argument to the app. 

You do not need the square brackets to use the argument.

**Reply by MeteorTony** [reply]
> `[--unicode]` is to indicate that you may optionally provide the `--unicode` argument to the app.
> 
> You do not need the square brackets to use the argument.

Yea I just tried this, it still gave the same error.

**Reply by Derppening** [reply]
Your current working directory is in "artifacts". The puzzle file is in the parent directory, so you need to specify "../puzzles/00-unicode-test.game" instead.

**Reply by MeteorTony** [reply]
thanks it works
